src/components/CheckIn/AddPieceModal.js

```diff
@@ -47,20 +52,68 @@ const footer = (close, save, checkIn) => (
   </ModalFooter>
 );
 
+const FIELD_ATTRS_FOR_REQUIRED_LOCATION = {
+  required: true,
+  validate: Required,
+};
+
 class AddPieceModal extends Component {
   static propTypes = {
+    change: PropTypes.func.isRequired,
```

Comment:
`change` what?

Reply:
change and dispatch is redux-form's props )